apps/wallet-mobile/src/features/Exchange/useCases/CreateExchangeOrder/SelectBuyOrSell/SelectBuyOrSell.tsx

```diff
@@ -24,15 +28,14 @@ export const SelectBuyOrSell = () => {
       <ButtonActionGroup
```

Comment:
ps: for later, most of the tab selectors should be a navigation to keep state. (this is just a remark)

Reply:
I also think that the order of the inputs are upside down
since you first have to select the provider, and based on the provider, it can have sell/buy, only sell, or only buy.  "just a remark"

Reply:
this has been simplified after set:

- Banxa -> Buy
- Encryptus -> Sell

Let me know if you think it can be improved please